### DISCUSSION TITLE
Custom Sonarr/Radarr Servers per User

### DISCUSSION THREAD

**Discussion by TubaYogy** [sct/overseerr#3203] [Feature Requests]
Apologies if this has all ready been requested/Discussed.

As there is the flexibility to add multiple Sonarr/Radarr servers to the Services as well as dedicated 4K services, Would it be possible to apply specific "arr" services to individual Users/Groups?

I currently have two instances of the "arr" service within my environment and two Plex Servers (one is mine and free from crap) and one for my Friends who require(?) media that I would never watch.  It would be ideal that if I log in, any requests by default go to Sonarr 1, and if they log in, any requests by default are added to Sonarr 2 without having to select it as their server.  We all know that if users have to make a decision, Default is always the end result and usually not what you want.

**Comment by ZiIlaGit104** [comment]
I 2nd this and would add, default/limited quality/path per user option would be amazing!
